plugin-build/src/main/kotlin/io/sentry/android/gradle/tasks/SentryUploadProguardMappingsTask.kt

```diff
@@ -73,7 +74,8 @@ abstract class SentryUploadProguardMappingsTask : Exec() {
             "upload-proguard",
             "--uuid",
             uuid,
-            mappingsFile.get().toString()
+            // throws Value 'file collection' specified for property 'mappingFiles' cannot be converted to a file.
+            mappingFiles.first().absolutePath
```

Comment:
@cortinico ideas? can you give it a try? could not figure out, not an expert either :D